### PR TITLE
data processing

### PR DESCRIPTION
**Pull request by hjw0703**
data folder(raw data and processed data) uploaded

### CODE REVIEW

File: data/processed_data.csv [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+Lviv,Ukraine,350.29,15.31,122.25,85.5,59.87,106.92,322.52,219.46,389.17,1021.36,-673.49
+Novosibirsk,Russia,324.89,17.68,147.11,107.48,110.27,117.84,312.29,254.03,497.36,1175.4,-640.2
+Bursa,Turkey,170.58,43.26,145.14,108.94,64.11,95.75,172.12,228.58,505.99,546.91,-293.92
+Brussels,Belgium,868.9,122.55,292.16,242.32,107.01,197.57,343.32,792.8,2610.25,3289.04,436.41
+Jerusalem,Israel,945.89,95.45,269.63,308.39,156.94,222.23,401.27,645.45,2178.68,7229.86,-221.13
+Melbourne,Australia,1136.76,146.67,239.26,304.24,117.24,199.21,293.88,1425.47,3182.83,6138.7,745.56
+Perth,Australia,876.12,132.38,234.2,309.12,99.5,207.94,288.56,1356.74,2842.9,3917.46,695.08
+Sydney,Australia,1544.16,195.61,257.28,327.36,115.56,188.8,320.08,1583.95,3440.93,7872.15,492.08
+Alexandria,Egypt,129.21,14.41,114.68,100.91,56.61,40.39,270.05,63.17,186.89,549.26,-539.39
+Quito,Ecuador,336.85,73.31,159.55,151.16,122.93,90.87,308.65,315.36,600.99,1331.22,-642.34
```
Comment (Arushi282) [inline data/processed_data.csv]: Hi @hjw0703, 

Great work so far on the preprocessing, please review my comment [here](https://github.com/UBC-MDS/Cost_of_living_py/issues/4) in the Data Processing issue, would request you to consider this suggestion. Thank you!

Comment (hjw0703) [reply data/processed_data.csv]: Thanks for your advice, as car prices vary a lot between different makes and models, I added car price for a popular model in the world: Toyota corolla 1.6L for our future visualization.

File: Proposal.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Section 3: Research questions and usage scenarios
+
+Our project answers the broad research question of: “What is the average
+cost of living for a single fresh graduate in different cities in the
+world?”. Below we have provided a usage scenario of our product by a
+member of our target audience:
+
+Bella is a fresh graduate in Canada and her current marital status is
+single. She wants to seek for a job and settle down in some major cities
+in the world, but she concerns about the high living cost in big cities,
```
Comment (Affrin101) [inline Proposal.md]:
```suggestion
in the world, but she has concerns about the high living cost in big cities,
```

File: Proposal.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+including housing, grocery, transportation, entertainment, shopping and
+so on. In order to help her get a general idea of how much the living
```
Comment (Affrin101) [inline Proposal.md]:
```suggestion
others. In order to help her get a general idea of how much the living
```

**Review by Affrin101 — CHANGES_REQUESTED** [review]
Hey @hjw0703 , the processed file looks great so does the Proposal.I have suggested very minor grammatical changes and then this  P can be merged. Kindly have a look at the suggestions and commit them if you feel the are fine.

**Review by Affrin101 — APPROVED** [review]

**Pull request merged**